pwd.txt

```diff
@@ -0,0 +1,2 @@
+kFZH7ebgmeVznzpA
```

Comment:
Not even sure why the pwd.txt file was even included in my local branch as I have the MONGO_URI set up in my .bash_profile file. Is having it in my .bash_profile vs .bashrc file a problem?
Also I deleted the pwd.txt file and subsequently issued the git rm --cached pwd.txt command which failed with error message "fatal: pathspec 'pwd.txt' did not match any files". I'm assuming that it failed because I deleted the file before issuing the ensuing command. Is there any remediation that I need to do now to clear it from my cache.

With regards to the resetting the MongoDb password, are there any specific instructions that you can point me to for doing this. The MongoDB instructions that you provided in the project were for setting up the mongoDb stuff....so I don't think I want to repeat those. I found other references to resetting mongodb passwords but I'm not sure it is referring to the same type of password...looks more like an end user password.  So any pointers or instructions would be helpful. 

Cheers,
Jet

Reply:
Awaiting input...hoping there is a simple way to change Mongo_DB password.